src/screens/NewsScreen/Search.js

```diff
@@ -82,21 +82,21 @@ const styles = StyleSheet.create({
     paddingTop: 0,
     paddingBottom: 0,
     alignSelf: 'center',
-    color: COLORS.gray1
+    color: COLORS.blackgrey
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The update from `COLORS.gray1` to `COLORS.blackgrey` is consistent with the PR's goal of enhancing the color theme. However, without knowing the exact colors represented by these variables, it's hard to say whether this change is an improvement or not.
```diff
-    color: COLORS.gray1
+    color: COLORS.blackgrey
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->